src/en/developer-testing.md

```diff
@@ -201,3 +201,74 @@ following:
 ```
     self.deployment.log.debug("Some debug message here.")
 ```
+
+## Unit testing a layered charm
+
+Amulet is a mature tool for deploying and testing a charm in a test environment.
+
+For layered charms, it is often desirable to be able to run some tests before the charm has been built, however. For example, you may wish to run unit tests as you write your code. Waiting for the charm to build so that amulet can run tests on it would introduce unnecessary delays into the unit testing cycle. What follows are some best practices for writing unit tests for a layered charm.
+
+#### Create a separate tox ini file
+
+You will usually want to create a second .ini file for tox, along with a separate requirements file, so that requirements for your unit tests don't clobber the requirements for Amulet tests. You might call this file `tox_unit.ini`, and put the following inside of it:
+
+```
+[tox]
+skipsdist=True
+envlist = py34, py35
+skip_missing_interpreters = True
+
+[testenv]
+commands = nosetests -v --nocapture tests/unit
+deps =
+    -r{toxinidir}/unit_test_requirements.txt
+    -r{toxinidir}/wheelhouse.txt
+
+setenv =
+    PYTHONPATH={toxinidir}/reactive:{toxinidir}/lib
```

Comment:
this can actually cause some collisions. I've found that removing reactive from PTYHONPATH and just doing `from reactive import $layer` works for unit tests since it's all python3